src/api/paths.js

```diff
@@ -0,0 +1,16 @@
+const paths = {
```

Comment:
I still don't really like this solution. Re-exporting the functions allows us the most flexibility in the future if (for example) we need to change the implementation for one resource. It's also almost the exact same amount of code.

In a better language, this could be done with rich enums but javascript doesn't have that.